web/api/v1/api.go

```diff
@@ -148,9 +149,16 @@ func (api *API) Register(r *route.Router) {
 	r.Get("/alertmanagers", instr("alertmanagers", api.alertmanagers))
 }
 
+type queryStats struct {
```

Comment:
@juliusv Hmm, but this moves the API specification outside of the API package. A change to the util package naming will also change the API now.